tracks/ruby/exercises/series/mentoring.md

```diff
@@ -0,0 +1,49 @@
+New concepts: Iterating over an array (preferably with `each_cons`, avoid `each_with_index`); `.chars` to split a string; (private) `attr_reader`; raising an Argument Error; inline if-statement 
+
+### Minimal solution for approval
+
+```ruby
+
+class Series
+
+  def initialize(serie)
+    @serie = serie
+  end
+
+  def slices(span)
+    raise ArgumentError if span > serie.size
+    serie.chars.each_cons(span).map {|slice| slice.join}
+  end
+
+  private
+  attr_reader :serie
+end
+```
+
+### Reasonable variants
+- Do the splitting in the initializer: 
+  ```
+    @numbers = serie.chars
+  ```
+
+### General 
+- The Instructions point to a beginner friendly [explanation of iterating in Ruby:](http://jeromedalbert.com/ruby-how-to-iterate-the-right-way/)
+and to the rubydocs [Enumerable](https://ruby-doc.org/core/Enumerable.html) with a hint that should make finding `each_cons` easier. 
+
+
+### Talking points
+- `each_cons` instead of an iterator `with_index`: In Ruby, you rarely have to write iterators that need to keep track of the index. Enumerable has powerful methods that do that for us.
+- `chars`: Use Ruby's built-in methods instead of implementing it yourself: `chars` instead of `split('')`   
```

Comment:
The comment about using the built in methods doesn't seem to apply here.  That seems to be an aside from what the lead is indicating,

`chars`: Use instead of `split('')`